src/messaging/messaging-types.ts

```diff
@@ -60,6 +61,10 @@ export interface Notification {
   body?: string;
 }
 
+export interface FcmOptions {
```

Comment:
I think there should be code in place to validate the analytics label in each of these structs. Similar to how we validate other fields during json marshaling.

Reply:
Ok done